grabbit/src/test/groovy/com/twcable/grabbit/client/GrabbitConfigurationSpec.groovy

```diff
@@ -25,6 +25,125 @@ import spock.lang.Unroll
 class GrabbitConfigurationSpec extends Specification {
 
     @Unroll
+    def "Should return exclude paths"() {
+        when:
+        def expectedOutput = ["/content/a/b/d", "/content/a/b/e/ab", "/content/a/b/q/fg"] as Collection<String>
+        def actualOutput = GrabbitConfiguration.create(input)
+
+        then:
+        actualOutput instanceof GrabbitConfiguration
+
+        actualOutput.pathConfigurations.excludePaths.get(0) == expectedOutput
+
+        where:
+        input << [
+                """
+            {
+                "serverUsername" : "admin",
+                "serverPassword" : "admin",
+                "serverHost" : "localhost",
+                "serverPort" : "4503",
+                "deltaContent" : false,
+                "pathConfigurations" :  [
+                    {
+                        "path" : "/content/a/b",
+                        "excludePaths" : ["d", "e/ab", "q/fg"],
+                        "workflowConfigIds" : []
+                    }
+                ]
+			}
+			"""
+        ]
+    }
+
+    @Unroll
+    def "Should fail to return exclude paths"() {
```

Comment:
Not a very descriptive feature name. Maybe something more along the lines of `"throws exception for non-relative exclude paths"`